### ISSUE TITLE
Support p4 move/rename, copy, and delete

### ISSUE THREAD

**Issue by unional** [mattsawyer77/atom-perforce#28]
Hi, first of all, I would like to thank you for creating this.  Without it, I simply won't have a chance to use atom for the project that I'm working on.

To make it useful to me (and my team), I need to have these additional commands available, especially move/rename.  I would definite want to contribute, but I don't have any experience in atom API.  Do you plan on adding move/rename anytime soon?

Best

**Comment by mattsawyer77** [comment]
@unional I'd like to implement features like this, but don't think Atom has nice idiomatic way of building features that require user input. I've been following https://github.com/atom/atom/issues/5756 to see what the way forward looks like. I'll leave this request open for now.

**Comment by PhiLhoSoft** [comment]
While I understand the request (it works in Eclipse, after all), I also understand it might be not trivial to do.
Personally, I just use P4V (the excellent visual client) for such operations, and more like file history, time-lapse view, etc. Actually, I prefer to use these latter features in P4V rather than in Eclipse. There is little point in trying to cram everything in one (external) tool.

**Comment by unional** [comment]
The main reason for this request is to capture the action done inside atom, as you can rename/move/delete from easily within atom (and very natural to do, just drag and drop or F2/Enter).

Instead of opening p4v for those operation.  File history, etc, are completely natural to do inside p4v.

**Comment by PhiLhoSoft** [comment]
Yes. As I wrote, I understand the need, and I find it useful. It is, somehow, even more crucial for Perforce because unlike Git, it won't automatically reconcile such external operation with an internal one.
Note that it is true also for doing such operations with mv or in Windows Explorer or similar (unless you have some shell extension)..
My remark was just: it doesn't shock me that this feature is currently missing from the package, and I can live without (unlike the core functionality of add / edit).
But it is a nice to have feature...
